src/gui/shader.cpp

```diff
@@ -54,8 +59,8 @@ load_shaders(
         fragment_shader_stream.close();
     } else {
         LOG_ERROR(
-            logger, "Cannot open {}. Are you in the right directory?",
-            fragment_file_path
+            logging::opengl_logger, "Cannot open {}. Are you in the right directory?",
+            fragment_file_path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.